src/providers/azure-ad-b2c.js

```diff
@@ -1,5 +1,7 @@
-export default function AzureADB2C(options) {
-  const tenant = options.tenantId ? options.tenantId : "common"
+export default (options) => {
```

Comment:
Could you make this a named default export function again? Goes together with auto-generating the index.js file